### PR TITLE
React UI - POC

### PR DESCRIPTION
**Pull request by ethanlaj**
This will split the files between api and client, making it easier for development.

Python app correctly serves the built index.html file in the client/dist directory. React components are displayed correctly

### CODE REVIEW

File: api/interfaces/react_ui.py
Comment (Klinefelters) [inline api/interfaces/react_ui.py:1]: Docstrings and comments need to be added. Either in this PR, or open an issue to look over all docstrings and comments.

File: api/models/frc7ng480.pt
Comment (Klinefelters) [inline api/models/frc7ng480.pt:1]: The models folder needs to be reviewed. This should also probably be a new issue.

**Comment by Klinefelters** [comment]
In general, the following needs to be done before this PR can be merged with main:

1. Install, run and dev-run scripts should be added, or a new issue for this should be made
2. Comments on current code should be resolved, mostly about new issues and such
3. The frontend has a bug where when switching tabs it resets the sliders. This should be fixed

File: api/models/frc7ng480.pt
Comment (Klinefelters) [reply api/models/frc7ng480.pt:1]: Made a new issue

File: api/interfaces/react_ui.py
Comment (ethanlaj) [reply api/interfaces/react_ui.py:1]: This should be added in this PR. We shouldn't make a habit of opening new issues for problems created by updating the code

**Review by Klinefelters — CHANGES_REQUESTED** [review]
Final changes...

Since this change adds dependencies it needs to be tested and installation instructions on the wiki need to be updated after release.

File: api/interfaces/react_ui.py
Comment (ethanlaj) [reply api/interfaces/react_ui.py:1]: Added comments

**Comment by ethanlaj** [comment]
@Klinefelters Trello ticket created to create those scripts

https://trello.com/c/3lgjTQ8L/16-add-scripts-to-root-directory

**Review by Klinefelters — APPROVED** [review]

**Pull request merged**